src/platform_impl/windows/monitor.rs

```diff
@@ -226,7 +226,7 @@ impl MonitorHandle {
         // EnumDisplaySettingsExW can return duplicate values (or some of the
         // fields are probably changing, but we aren't looking at those fields
         // anyway), so we're using a BTreeSet deduplicate
-        let mut modes = BTreeSet::<RootVideoModeHandle>::new();
+        let mut modes = Vec::new();
         let mod_map = |mode: RootVideoModeHandle| mode.video_mode;
```

Comment:
This code doesn't need to mention `RootVideoModeHandle` any more, and can be simplified even further

Reply:
Similarly for some of the other backends.

Reply:
Done!